timbermill-java/timbermill-client/src/main/java/com/datorama/oss/timbermill/pipe/TimbermillServerOutputPipe.java

```diff
@@ -128,9 +132,10 @@ private void sendEvents(EventsWrapper eventsWrapper) throws IOException {
     }
 
     private void sendEventsOverConnection(HttpURLConnection httpCon, byte[] eventsWrapperBytes) throws IOException {
-		try (OutputStream os = httpCon.getOutputStream()) {
-			os.write(eventsWrapperBytes, 0, eventsWrapperBytes.length);
-		}
+        try (OutputStream httpConOS = httpCon.getOutputStream();
```

Comment:
support gzip/non-gzip types on client side. mainly for testing purposes for backward-compatibility